packages/underscore-redirects/src/astro.ts

```diff
@@ -125,15 +138,15 @@ export function createRedirectsFromAstroRoutes({
  * /team/articles/*
  * With stars replacing spread and :id syntax replacing [id]
  */
-function generateDynamicPattern(route: RouteData) {
+function generateDynamicPattern(route: RouteData, forTarget = false) {
 	const pattern =
 		'/' +
 		route.segments
 			.map(([part]) => {
 				//(part.dynamic ? '*' : part.content)
 				if (part.dynamic) {
 					if (part.spread) {
-						return '*';
+						return forTarget ? ':splat': '*';
```

Comment:
As per:
https://developers.cloudflare.com/pages/configuration/redirects/#splats and https://docs.netlify.com/routing/redirects/redirect-options/#splats